app/metrics.yaml

```diff
@@ -130,3 +130,130 @@ settings_screen:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-08-17"
+
+tracking_protection:
+  toolbar_shield_clicked:
+    type: counter
+    description: |
+      A counter that indicates how many times a user has opened
+      the tracking protection settings panel from the toolbar.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5057
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5163
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-17"
+
+  tracking_protection_changed:
+    type: event
+    description: |
+      The user has changed the setting for enhanced tracking protection.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5057
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5163
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-17"
+
+  has_ever_changed_etp:
+    type: boolean
+    description: |
+      The user has changed the setting for enhanced tracking protection
+      at least once.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5057
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5163
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-17"
+
+  tracker_setting_changed:
+    type: event
+    description: |
+      The user has changed the advertising tracker protection state.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5057
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5163
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-17"
+    extra_keys:
+      source_of_change:
+        description: The source of interaction, "Panel" or "Settings"
+        type: string
+      tracker_changed:
+        description: |
+          The tracker changed, "Advertising", "Analytics", "Social", "Content"
+        type: string
```

Comment:
Curious why you don't also include the value that the setting was changed to here?

Reply:
I added  `isEnabled` extra.